misc/package/Data Files/MWSE/core/mcm/main.lua

```diff
@@ -167,6 +269,8 @@ local function onClickModConfigButton()
 
 	local menu = tes3ui.findMenu("MWSE:ModConfigMenu")
 	if (not menu) then
+		-- load the list of favorite mods
```

Comment:
Was it decided later in development that loading won't be done here?

Reply:
yeah, i figured it would be better to just load once when the game initialized and then forgot to delete the comment